### DISCUSSION TITLE
Pass additional env value to collabora container

### DISCUSSION THREAD

**Discussion by rvboris** [nextcloud/all-in-one#1920] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 

I have an issue with making work collabora because any office file links resolve the incorrect domain name, and it looks like this:

`https://192.168.1.100/browser/719ae9b/cool.html?WOPISrc=https%3A%2F%2Fnc.atlanta.keenetic.link%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F376620_oczt1pzbeh34&title=test.ods&lang=en&closebutton=1&revisionhistory=1`

There's an IP address. It's a host IP address.

### Describe the solution you'd like

To make it works we need just pass [server_name](https://sdk.collaboraonline.com/docs/installation/CODE_Docker_image.html) env value to collabora container

### Additional context

I am using ddns from my router, and collabora can't resolve it correctly.

**Comment by szaimen** [comment]
Hi, are you using AIO behind a reverse proxy or standalone?

**Reply by rvboris** [reply]
Hi, I use it standalone, only with ddns from the router.

**Comment by szaimen** [comment]
So there is no proxy in front of AIO? I am asking because I cannot reproduce the issue on my test server. 

Also, can you first follow https://github.com/nextcloud/all-in-one/discussions/1358 before we continue with a possible fix to this?

**Reply by rvboris** [reply]
Without any proxies, just a fresh install. 

yes, I saw that thread, it little bit different problem, because collabora itself is accessible out of the box, and I see a green mark in settings, the problem is only with links.

**Reply by szaimen** [reply]
> the problem is only with links.

What links do you mean? So internally it works but if you open a share link it doesnt?

**Reply by rvboris** [reply]
> What links do you mean? So internally it works

Yes it works, but links of files contains wrong host name, not sharing link, all of documents.

**Comment by szaimen** [comment]
So basically collabora works for you and only the logs display a wrong domain?

**Reply by rvboris** [reply]
> So basically collabora works for you and only the logs display a wrong domain?

WOPI links to any collabora files has wrong domain to fix this issue i add env value directly to collabora container.

**Comment by szaimen** [comment]
I see. I can still not reproduce the issue but let me add this variable if it should be required in your case.

**Comment by szaimen** [comment]
Just a short question: should it be `server_name=yourncdomain.com` or e.g. `server_name=https://yourncdomain.com` or e.g.  `server_name=https://yourncdomain.com:443` ?

**Comment by szaimen** [comment, marked as answer]
Added with https://github.com/nextcloud/all-in-one/pull/1939

**Reply by szaimen** [reply]
This is now released with v4.3.2 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
